synfig-studio/src/gui/widgets/widget_timetrack.cpp

```diff
@@ -198,6 +198,47 @@ bool Widget_Timetrack::move_selected(synfig::Time delta_time)
 	return ok;
 }
 
+ void Widget_Timetrack::interpolate_selected(synfig::Interpolation type)
+{
+	synfig::Waypoint::Model model;
+	model.set_before(type);
+	model.set_after(type);
+	std::vector<WaypointItem*> selection = waypoint_sd.get_selected_items(); //storing selected waypoint items
+	std::vector<std::set<synfig::Waypoint, std::less<synfig::UniqueID> >> accumilated_selection_set;
```

Comment:
1. typo: accumulated
2. why do you need an accumulated set?

Reply:
it seems like I used it to store the fetched waypoint set and iterate through them -cant remember why-, but yes its not necessary we can use `waypoint_set_new` right away, better to remove it.